### PR TITLE
Aks helm

### PR DESCRIPTION
**Pull request by chris-moreton**
Initial attempt to get Case Loader using AKS & Helm Charts

### CODE REVIEW

File: charts/sscs-case-loader/values.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      - sftp-port
+      - gaps2-service-sftp-private-key
+  environment:
+    CORE_CASE_DATA_JURISDICTION_ID: SSCS
+    IDAM_OAUTH2_CLIENT_ID: "sscs"
+    CORE_CASE_DATA_CASE_TYPE_ID: Benefit
+    IDAM_OAUTH2_REDIRECT_URL: "https://evidence-sharing-preprod.sscs.reform.hmcts.net"
+    IDAM_S2S_AUTH: "http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
+    CORE_CASE_DATA_API_URL: "http://ccd-data-store-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
+    IGNORE_CASES_BEFORE_DATE: 1017-07-01
```
Comment (timja) [inline charts/sscs-case-loader/values.yaml]: is 1017 right?

File: src/main/resources/bootstrap.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        idam-redirect: IDAM_OAUTH2_REDIRECT_URL
+        idam-sscs-oauth2-client-secret: IDAM_OAUTH2_CLIENT_SECRET
+        idam-oauth-user: IDAM_OAUTH2_CLIENT_ID
+        ccd-jid: CORE_CASE_DATA_JURISDICTION_ID
+        ccd-tid: CORE_CASE_DATA_CASE_TYPE_ID
+        sftp-dir: GAPS2_SFTP_DIR
+        sftp-user: GAPS2_SFTP_USER
+        sftp-host: GAPS2_SFTP_HOST
+        sftp-port: GAPS2_SFTP_PORT
+        gaps2-service-sftp-private-key: GAPS2_KEY_LOCATION
```
Comment (timja) [inline src/main/resources/bootstrap.yaml]: missing application insights key
https://github.com/hmcts/draft-store/blob/master/src/main/resources/bootstrap.yaml#L8

File: Jenkinsfile_CNP [hunk 3]
```diff
@@ -41,13 +41,21 @@ static LinkedHashMap<String, Object> secret(String secretName, String envVar) {
...
+  enableDockerBuild()
+  installCharts()
   loadVaultSecrets(secrets)
+  enableSlackNotifications('#sscs-tech')
+  enableAksStagingDeployment()
+  nonServiceApp()
+
+  before('akschartsinstall') {
+    echo "Setting up data for functional tests"
+    gradle(functionalPreDeploy)
```
Comment (timja) [inline Jenkinsfile_CNP]: you should be using gradleBuilder:
https://github.com/hmcts/bulk-scan-processor/blob/master/Jenkinsfile_CNP#L64
`GradleBuilder builder = new GradleBuilder(this, product)`
`builder.gradle('functionalPreDeploy')`

File: Jenkinsfile_CNP [hunk 4]
```diff
@@ -41,6 +41,8 @@ static LinkedHashMap<String, Object> secret(String secretName, String envVar) {
   ]
 }
 
+GradleBuilder builder = new GradleBuilder(this, product)
```
Comment (timja) [inline Jenkinsfile_CNP]: requires import as well https://github.com/hmcts/bulk-scan-processor/blob/master/Jenkinsfile_CNP#L5

Comment (chris-moreton) [reply Jenkinsfile_CNP]: Thanks @timja - we're still getting the same error of..

ERROR: JAVA_HOME is not set and no 'java' command could be found in your PATH.

File: charts/sscs-case-loader/values.yaml [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+      - idam-redirect
+      - idam-sscs-oauth2-client-secret
+      - idam-oauth-user
+      - ccd-jid
+      - ccd-tid
+      - sftp-dir
+      - sftp-user
+      - sftp-host
+      - sftp-port
+      - gaps2-service-sftp-private-key
```
Comment (timja) [inline charts/sscs-case-loader/values.yaml]: Missing the app insights key here

File: charts/sscs-case-loader/values.preview.template.yaml [hunk 6]
```diff
@@ -0,0 +1,4 @@
+job:
+  image: ${IMAGE_NAME}
+  ingressHost: ${SERVICE_FQDN}
+  aadIdentityName:
```
Comment (timja) [inline charts/sscs-case-loader/values.preview.template.yaml]: probably want to duplicate the vault definition here to remove the app insights key when you add it

**Pull request merged**